### ISSUE TITLE
Fields on abstract models aren't computed on save

### ISSUE THREAD

**Issue by mobiware** [netzkolchose/django-computedfields#12]
If I define an `@computed` field on an Abstract model and then have a Django model inherit that abstract model, the field is not computed on save.

Example:
```
class Abstract(ComputedFieldsModel):
    class Meta:
        abstract = True

    a = models.IntegerField(default=0)
    b = models.IntegerField(default=0)

    @computed(models.IntegerField(default=0))
    def c(self):
        return self.a + self.b


class Concrete(Abstract):
    d = models.IntegerField(default=0)
```

In the above example, `c` doesn't get computed, it remains equal to `0`:
```
In [1]: obj = Concrete.objects.create(a=2, b=3)                                                                                                                                                                                             

In [2]: obj.c                                                                                                                                                                                                                               
Out[2]: 0
```

**Comment by jerch** [comment]
Thx for spotting this and the PR. Will have to make a few more tests before I can merge it, as the code contains a few edge case assumptions while walking the relations, to make sure they keep working across abstract definitions.

**Comment by mobiware** [comment]
OK, what extra tests do we need ?

**Comment by jerch** [comment]
I think we should postpone it until #15 got merged (which is a pretty bad flaw in the logic, kinda overlooked it in the first place).

After that I think we should add a test case for proxied cf fields for abstract as well, similar to: https://github.com/netzkolchose/django-computedfields/blob/6e050a7e35712adc85ff4e480e0af00a438291d5/example/test_full/models.py#L163-L189

just to make sure, that cf fields still can be depended from pure abstract as well and derived correctly.

**Comment by mobiware** [comment]
Sounds good, I'll rebase and add the tests once the other PR is merged.

**Comment by jerch** [comment]
@mobiware Just merged the other PR, thus feel free to continue. Would be great to have this working.

**Comment by mobiware** [comment]
@jerch I've just push additional tests to my PR
